packages/builder/src/components/common/users/PasswordRepeatInput.svelte

```diff
@@ -1,40 +1,52 @@
 <script>
-  import { Layout, Input } from "@budibase/bbui"
+  import { FancyForm, FancyInput } from "@budibase/bbui"
   import { createValidationStore, requiredValidator } from "helpers/validation"
 
   export let password
+  export let passwordForm
   export let error
 
+  const validatePassword = value => {
+    if (!value || value.length < 12) {
```

Comment:
Is this 12 right? We use 8 by default on the backend, and it can be specified case by case. Should we use the same `PASSWORD_MIN_LENGTH` value?